### PR TITLE
small fix for #3 memory leak problem

### PR DESCRIPTION
**Pull request by andrei-diaconu**
I sort of needed this fix, and this works for my case. I haven't spent too much time thinking about nested structures and arrays, as my case is very similar to the one described in the issue, and now everything works perfectly with no memory leaks.

I want to stress again, that this solution might not be correct, but it solves my problems until the correct one is created.

### CODE REVIEW

File: src/tpl.c [hunk 1]
```diff
@@ -253,7 +253,7 @@ static const struct tpl_type_t tpl_types[] = {
 static int tpl_oops(const char *fmt, ...) {
     va_list ap;
     va_start(ap,fmt);
-    vfprintf(stderr,fmt,ap);
+    vprintf(fmt,ap);
```
Comment (tbeu) [inline src/tpl.c]: This rather should go to a separate pull request.

Comment (andrei-diaconu) [reply src/tpl.c]: You are right. I didn't want this change to be part of the pull request for the memory leak issue.

Probably my mistake was creating the pull request on my master branch (I don't work with pull requests). I stopped supporting this pull request because I am not sure this is the correct solution for all cases, as explained above. But for my use case I need the change.

I also need to avoid stderr for my case. I want tpl.lib to be usable in VS versions both before and after 2015. But I do not want this change to make it into the "official" version.

If you don't mind, I would close this pull request.

Comment (tbeu) [reply src/tpl.c]: Yes, having static libs compiled with VS < 2015 and linking later with VS 2015 is pain. I also needed to deal with it, see https://github.com/tbeu/ExternData/blob/master/ExternData/Resources/Include/msvc_compatibility.h, but relied on not using stderr at all.
